d2l-icon-button-shared-styles.html

```diff
@@ -1,67 +0,0 @@
-<link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../d2l-offscreen/d2l-offscreen-shared-styles.html">
```

Comment:
I think this is the last reference to the `offscreen` component, so it can be removed from `bower.json`?